taxonomies/ingredients_processing.txt

```diff
@@ -2125,48 +2343,48 @@ nl: ongezouten
 pl: bez soli, niesolone
 pt: sem sal, não salgado
 
-en: sweetened
-bg: подсладено, подсладен, подсладена, подсладени
-de: gesüßt, gesüßte
-es: endulzado, endulzada, endulzados, endulzadas
-fi: makeutettu, makeutetut
-hr: zaslađene
-hu: édesített
-it: addolcito, addolciti, addolcita, addolcite
-nl: gezoet, gezoete
-pl: słodzony, słodzona, słodzone, słodzonych, słodzonej, słodzonego
-pt: adoçado, adoçada, adoçados, adoçadas, adocicado, adocicada, adocicados, adocicadas
-sv: sötad
+# sweetened is not a processing as it means an extra ingredient. 
+# It changes an ingredients into a 'compound' ingredient.
```

Comment:
I think this is going beyond the scope of the PR. If this is a distinction we want to make going forward, I think it deserves its own PR with its own discussion. It feels like overreach to group it in with “move processing from ingredients to processing”.